extensions/replication/tasks/UpdateReplicationStatus.js

```diff
@@ -246,6 +246,7 @@ class UpdateReplicationStatus extends BackbeatTask {
         }
         const versionId =
             sourceEntry.getReplicationSiteDataStoreVersionId(site);
+        entry.setOriginOp('');
```

Comment:
For my understanding can you explain why we set an empty origin op ?

Reply:
This is to avoid double notifications essentially, as keeping the original field value would trigger a new notification on update, but maybe we could/should set an actual value here instead like `s3:ReplicateObject` (if that's your original question).

Reply:
`s3:ReplicateObject` is not AWS standard.. 
here are the only standard notifications for replication: 
```
s3:Replication:*
s3:Replication:OperationFailedReplication
s3:Replication:OperationMissedThreshold
s3:Replication:OperationReplicatedAfterThreshold
s3:Replication:OperationNotTracked
```

I don't see any that can fit our need here..

Can we introduce a non standard notification for that case (should we ?) ?